шахматы/шахматы/Program.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace шахматы
+{
+    internal class Program
+    {
+        static void Main(string[]args)
+        {
+            Console.WriteLine("Введите позицию белого коня");
+            var WhiteKnightPosition = Console.ReadLine();
+            Console.WriteLine("Введите позицию черного коня");
+            var BlackKnightPosition = Console.ReadLine();
+
+            if (!CheckPosition(WhiteKnightPosition) || !CheckPosition(BlackKnightPosition))
+            {
+                Console.WriteLine("Так ставить фигуры нельзя. ");
+                Console.ReadKey();
+
```

Comment:
Здесь программа должна закончить работу
```suggestion
                Console.ReadKey();
                return;
```